### PR TITLE
[TR] PIM-3537: Add a nice error page

### PR DESCRIPTION
**Pull request by sumbobyboys**
| Q                    | A
| -------------------- | ---
| Bug fix?             | Y
| New feature?         | N
| BC breaks?           | N
| CI currently passes? | Y
| Tests pass?          | Y
| Scenarios pass?      | Y
| Checkstyle issues?*  | N
| PMD issues?**        | N
| Changelog updated?   | N
| Fixed tickets        | PIM-3537
| DB schema updated?   | N
| Migration script?    | N
| Doc PR               |

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/EventListener/KernelExceptionListener.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    {
+        $exception = $event->getException();
+
+        $response = new Response();
+
+        if ($exception instanceof HttpExceptionInterface) {
+            $response->setStatusCode($exception->getStatusCode());
+            $response->headers->replace($exception->getHeaders());
+        } else {
+            $response->setStatusCode(500);
```
Comment (jjanvier) [inline src/Pim/Bundle/EnrichBundle/EventListener/KernelExceptionListener.php]: why not $response->setStatusCode($exception->getStatusCode()); ?

**Comment by jjanvier** [comment]
GTM

File: src/Pim/Bundle/EnrichBundle/EventListener/KernelExceptionListener.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+     * @param EngineInterface $templating
+     */
+    public function __construct(EngineInterface $templating)
+    {
+        $this->templating = $templating;
+    }
+
+    /**
+     * Manage kernel exception
+     * @param  GetResponseForExceptionEvent $event
```
Comment (grena) [inline src/Pim/Bundle/EnrichBundle/EventListener/KernelExceptionListener.php]: Double spaces after `@param`

File: src/Pim/Bundle/EnrichBundle/EventListener/KernelExceptionListener.php [hunk 3]
```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Pim\Bundle\EnrichBundle\EventListener;
+
+use Pim\Bundle\EnrichBundle\Exception\ObjectNotFoundHttpException;
```
Comment (grena) [inline src/Pim/Bundle/EnrichBundle/EventListener/KernelExceptionListener.php]: Not used here apparently

**Comment by grena** [comment]
GTM

**Pull request merged**